authors/apps/article_tags/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from .models import ArticleTag
+
+
+class ArticleTagSerializer(serializers.ModelSerializer):
```

Comment:
please add a doc string to describe what the class does

Reply:
I have added a doc string.. Thanks for the observation